expression/builtin_miscellaneous.go

```diff
@@ -204,7 +204,58 @@ type builtinInetAtonSig struct {
 
 // See https://dev.mysql.com/doc/refman/5.7/en/miscellaneous-functions.html#function_inet-aton
 func (b *builtinInetAtonSig) eval(row []types.Datum) (d types.Datum, err error) {
-	return d, errFunctionNotExists.GenByArgs("INET_ATON")
+	d.SetNull()
+	args, err := b.evalArgs(row)
+	if err != nil {
+		return d, errors.Trace(err)
+	}
+	arg := args[0]
+	if arg.IsNull() {
+		return d, nil
+	}
+	s, err := arg.ToString()
+	if err != nil {
+		return d, errors.Trace(err)
+	}
+
+	c := byte('.') // if len(s) == 0, return NULL
```

Comment:
May be we can make it more elegant.

1. We can remove `c` and `n`, and check `len(s) == 0 || s[len(s)-1] == '.'` out of the for-loop.

2. 
``` go
var(
    byteResult, sum uint64
    dotCnt int
)
```

3.  put the comment to line242.